src/checkers/inference/dataflow/InferenceValue.java

```diff
@@ -221,4 +222,22 @@ private TypeMirror getGlbType(final CFValue other, final CFValue backup) {
 
         return underlyingType;
     }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder("InferenceValue{annotations=");
+        Slot slot = getEffectiveSlot(this);
+        if (slot.isConstant()) {
+            sb.append(((ConstantSlot) slot).getValue());
```

Comment:
Instead of using `toString` from `AnnotationMirror`, it's better to use the `analysis.getTypeFactory().getAnnotationFormatter()` to properly format the annotation.
If possible, fix the implementation of `ConstantSlot.toString` to return the properly-formatted string.